docs/reference/sql/functions/search.asciidoc

```diff
@@ -8,6 +8,109 @@ when the `MATCH` or `QUERY` predicates are being used.
 Outside a, so-called, search context, these functions will return default values
 such as `0` or `NULL`.
 
+[[sql-functions-search-match]]
+==== `MATCH`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+MATCH(field_exp<1>, constant_exp<2>[, options]<3>)
+--------------------------------------------------
+
+*Input*:
+
+<1> field(s) to match
+<2> matching text
+<3> additional parameters; optional
+
+.Description:
+
+A full-text search option, in the form of a predicate, available in {es-sql} that gives the user control over powerful <<query-dsl-match-query,match>>
+and <<query-dsl-multi-match-query,multi_match>> {es} queries.
+
+The first parameter is the field or fields to match against. In case it receives one value only, {es-sql} will use a `match` query to perform the search:
+
+["source","sql",subs="attributes,callouts,macros"]
+----
+include-tagged::{sql-specs}/docs/docs.csv-spec[simpleMatch]
+----
+
+However, it can also receive a list of fields and their corresponding optional `boost` value. In this case, {es-sql} will use a
```

Comment:
I would add a basic explanation on boosting does in this example for SQL folks that might be familiar with it.